.travis.yml

```diff
@@ -2,15 +2,33 @@ language: python
 cache:
   - pip
 
-python:
- - 2.7
- - pypy
+matrix:
+  include:
+    - python: 2.7
+      env: TOXENV=py27
+    - python: 3.4
+      env: TOXENV=py34
+    - python: 3.5
+      env: TOXENV=py35
+    - python: 3.6
+      env: TOXENV=py36
+    # 3.7 issue https://github.com/travis-ci/travis-ci/issues/9069
+    - python: 3.7
+      env: TOXENV=py37
+      sudo: true
+      dist: xenial
```

Comment:
Note: I would go straight to bionic, but there's [radio silence](https://travis-ci.community/t/ubuntu-18-04-1-lts-bionic-beaver/1270) from the Travis folks on that front.